assets/files.wxs

```diff
@@ -1649,6 +1649,9 @@
         <Component Id="cmp770700CA06564FB187D1CD2A79AA3DA8" Guid="{c244e7e1-3cb0-4880-aba5-e2be4fd78fbf}">
```

Comment:
@bergmeister, your last commit had 2 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Set-Service can accept a ServiceController as positional input
```powershell
Expected strings to be the same, but they were different.
String lengths are both 14.
Strings differ at index 0.
Expected: 'newdisplayname'
But was:  'testsetservice'
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 321
321:             $service.DisplayName | Should -BeExactly $newdisplayname
```
Get-WinEvent.Get-WinEvent can retrieve events.Get-WinEvent can use the simplest of filters
```powershell
Cannot convert argument "value", with value: "System.Diagnostics.Eventing.Reader.EventLogRecord", for "TryAdd" to type "System.Diagnostics.Eventing.Reader.EventLogRecord": "Cannot convert the "System.Diagnostics.Eventing.Reader.EventLogRecord" value of type "System.Diagnostics.Eventing.Reader.EventLogRecord" to type "System.Diagnostics.Eventing.Reader.EventLogRecord"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Diagnostics\Get-WinEvent.Tests.ps1: line 61
61:                 $testEventDict.TryAdd($te.Id, $te)

```

Reply:
It is worthwhile noting that the first Windows service test throws an `ObjectDisposedException` before it fails:
```
		Set-Service : Cannot access a disposed object.
Object name: 'ServiceController'.
At C:\Users\cberg\git\PowerShell\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1:318 char:25
+ ...       $script = { Set-Service $service -DisplayName $newdisplayname }
+                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (:) [Set-Service], ObjectDisposedException
+ FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.SetServiceCommand
```
![image](https://user-images.githubusercontent.com/9250262/59588305-31da2f00-90df-11e9-952d-be655ff6deae.png)

Reply:
@bergmeister Thanks for updating package versions!

Can you try to run Get-Service cmdlet manually in interactive session and see results?

Reply:
`Get-Service` on its own has no problem, it seems to be the interaction using `New-Service` and `Set-Service` that causes the failure later.

Reply:
It is expected behavior if we look line 2230 in Service.cs. It is not clear why we dispose ServiceController() before output.

Reply:
I do not found any changes in ServiceController.cs performed recently
https://github.com/dotnet/corefx/commits/60d8aa31a4c1ffa0e125bd7941aca171439a2f72/src/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceController.cs

It seems a root cause is in another place.
@SteveL-MSFT @daxian-dbw Have you any guesses?